src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeAnaylsisHelper.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.MSBuild;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.MSIdentity.CodeReaderWriter
+{
+    public class CodeAnaylsisHelper
+    {
+        private string _projectFilePath;
+        public CodeAnaylsisHelper(string projectFilePath)
+        {
+            _projectFilePath = projectFilePath;
+        }
+
+        private CodeAnalysis.Project? _codeAnalysisProject;
+        public CodeAnalysis.Project CodeAnalysisProject
+        {
+            get
+            {
+                if (_codeAnalysisProject == null)
+                {
+                    _codeAnalysisProject =  LoadCodeAnalysisProject().Result;
+                }
+                return _codeAnalysisProject;
+            }
+        }
+
+        private async Task<CodeAnalysis.Project> LoadCodeAnalysisProject()
```

Comment:
>LoadCodeAnalysisProject [](http://example.com/codeflow?start=49&length=23)

Async suffix to all async methods

Reply:
fixed and moved https://github.com/dotnet/Scaffolding/pull/1593